eth/ethconfig/config.go

```diff
@@ -248,17 +249,21 @@ const (
 
 func SyncModeByChainName(chain, syncCliFlag string) (SyncMode, error) {
 	if syncCliFlag == "full" {
+		log.Info("Syncmode", "type", FullSync)
 		return FullSync, nil
 	} else if syncCliFlag == "snap" {
+		log.Info("Syncmode", "type", SnapSync)
 		return SnapSync, nil
 	} else if syncCliFlag != "" {
 		return FullSync, fmt.Errorf("unexpected syncmode: %s, only next options are valid: %s, %s", syncCliFlag, FullSync, SnapSync)
 	}
 	switch chain {
```

Comment:
Why don't you print it in the place where `SyncModeByChain` is called? It would make more sense

Reply:
Done